Makefile.PL

```diff
@@ -137,6 +137,19 @@ CONTRIBUTORS.txt:
 
 END_CONTRIBUTORS
 
+    my $docker_make = <<'END_DOCKER';
+
+docker-build: dist
+	sudo docker build --tag zonemaster/ldns:local --build-arg version=$(VERSION) .
```

Comment:
I put sudo in there because the official documentation on root-less docker had rather strong warnings. When I looked back just now the warnings are much less dire than what I remembered. And the warnings don't seem to apply to our use case. I guess they've updated their documentation.

In any case I removed sudo. Anyone who isn't a member of the docker group should be able to work around the situation by running `sudo make docker-build`.